platform/android/java/editor/build.gradle

```diff
@@ -0,0 +1,63 @@
+// Gradle build config for Godot Engine's Android port.
+apply plugin: 'com.android.application'
+
+dependencies {
+    implementation libraries.kotlinStdLib
+    implementation libraries.androidxFragment
+    implementation project(":lib")
+}
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+    ndkVersion versions.ndkVersion
+
+    defaultConfig {
+        applicationId "org.godotengine.editor"
+        minSdkVersion versions.minSdk
+        targetSdkVersion versions.targetSdk
+    }
+
+    compileOptions {
+        sourceCompatibility versions.javaVersion
+        targetCompatibility versions.javaVersion
+    }
+
+    buildTypes {
+        release_debug {}
+    }
+
+    lintOptions {
+        abortOnError false
+        disable 'MissingTranslation', 'UnusedResources'
+    }
+
+    packagingOptions {
+        exclude 'META-INF/LICENSE'
+        exclude 'META-INF/NOTICE'
+
+        // 'doNotStrip' is enabled for development within Android Studio
+        if (shouldNotStrip()) {
+            doNotStrip '**/*.so'
+        }
```

Comment:
I think you can see in the build that it is only 64 mb, I don't think it hurt the apk that much

Reply:
It's not about the size but rather keeping the code clean and concise. We should strive to avoid [cargo cult programming](https://en.wikipedia.org/wiki/Cargo_cult_programming) by ensuring our code / logic only contains necessary elements.

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2